mkosi/__init__.py

```diff
@@ -7299,7 +7333,10 @@ def run_qemu(args: CommandLineArguments) -> None:
     if args.network_veth:
         if not ensure_networkd(args):
             # Fall back to usermode networking if the host doesn't have networkd (eg: Debian)
-            cmdline += ["-nic", "user,model=virtio-net-pci"]
+            fwd = ""
+            if args.ssh_port is not None:
+                fwd = f",hostfwd=tcp::{args.ssh_port}-:{args.ssh_port}"
```

Comment:
```suggestion
            fwd = f",hostfwd=tcp::{args.ssh_port}-:{args.ssh_port}"  if args.ssh_port is not None or ""
```
This should be within our linelength limit.